src/Gir/Generation/Method.cs

```diff
@@ -8,9 +8,14 @@ public partial class Method : INativeCallable
 
 		public void Generate (IGeneratable parent, IndentWriter writer)
 		{
-			if (Deprecated)
-				writer.WriteLine ($"[Obsolete (\"(Version: {DeprecatedVersion}) {DocDeprecated.Text}\")]");
-
+			if (!string.IsNullOrEmpty (Deprecated)) {
+				if (Deprecated == "0") {
```

Comment:
This check is redundant?

Reply:
`if (Deprecated == "1") -> obsolete version`
`else if (Deprecated != "0") -> obsolete`